src/prefect/client/orion.py

```diff
@@ -2036,6 +2037,42 @@ async def create_work_pool(
 
         return pydantic.parse_obj_as(WorkPool, response.json())
 
+    async def update_work_pool(
```

Comment:
Change this to a `workpoolUpdate`

Reply:
Actually, it doesn't make sense to have this take a workpool update instead of a workpool and optional, because we need the actual workpool information to set values if they aren't provided.